docs/data/base/components/checkbox/checkbox.md

```diff
@@ -9,108 +9,78 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/
 
 # Checkbox
 
-<p class="description">Checkboxes give users binary choices when presented with multiple options in a series.</p>
+<p class="description">Checkbox give users a binary choice between multiple options in a series.</p>
 
 {{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
 
 {{"component": "modules/components/ComponentPageTabs.js"}}
 
-## Introduction
+{{"demo": "UnstyledCheckboxIntroduction", "defaultCodeOpen": false, "bg": "gradient"}}
 
-The Checkbox component provides users with a checkbox for toggling a checked state.
+## Installation
 
-{{"demo": "UnstyledCheckboxIntroduction", "defaultCodeOpen": false, "bg": "gradient"}}
+Base UI components are all available as a single package.
 
-## Component
+<codeblock storageKey="package-manager">
 
-```jsx
-import * as Checkbox from '@base_ui/react/Checkbox';
+```bash npm
+npm install @base_ui/react
 ```
 
-### Anatomy
-
-The `Checkbox` component is composed of a root component and an indicator child component:
+```bash yarn
+yarn add @base_ui/react
+```
 
-```tsx
-<Checkbox.Root>
-  <Checkbox.Indicator />
-</Checkbox.Root>
+```bash pnpm
+pnpm add @base_ui/react
 ```
 
-The indicator can contain children, such as an icon:
+</codeblock>
 
-```tsx
-<Checkbox.Root>
-  <Checkbox.Indicator>
-    <MyCheckIcon />
-  </Checkbox.Indicator>
-</Checkbox.Root>
+Once you have the package installed, import the component.
+
+```ts
+import * as Checkbox from '@base_ui/react/Checkbox';
 ```
 
-The indicator conditionally unmounts its children when the checkbox is unchecked. For CSS animations, you can use the `keepMounted` prop to transition `visibility` and `opacity` for example:
+## Anatomy
+
+Checkbox is composed of two components.
+
+- `<Checkbox.Root />` renders a `<button>`.
+- `<Checkbox.Indicator />` renders a `<span>` for providing a visual indicator. You could place an icon inside this component.
 
 ```tsx
 <Checkbox.Root>
-  <Checkbox.Indicator keepMounted>
-    <MyCheckIcon />
-  </Checkbox.Indicator>
+  <Checkbox.Indicator />
 </Checkbox.Root>
 ```
 
-### Custom structure
+## Overriding default components
```

Comment:
It feels that this is a more advanced (and less commonly used) topic, so how about we move it further down?

We could use the following layout:
- Installation
- Anatomy
- component-specific...
- Overriding default components
- Accessibility